website/source/docs/auth/aws.html.md

```diff
@@ -0,0 +1,1006 @@
+---
```

Comment:
This is great docs.  It would have saved a lot of back and forth between myself and @jefferai :smile: